src/integration-test.ts

```diff
@@ -56,3 +65,38 @@ describe('Nativefier', () => {
     },
   );
 });
+
+describe('Nativefier upgrade', () => {
+  jest.setTimeout(300000);
+
+  test.each([
+    { platform: 'darwin', arch: 'arm64' },
+    { platform: 'darwin', arch: 'x64' },
+    { platform: 'linux', arch: 'arm64' },
+    { platform: 'linux', arch: 'armv7l' },
+    { platform: 'linux', arch: 'ia32' },
+    { platform: 'linux', arch: 'x64' },
```

Comment:
Any reason you don't test Windows?

Reply:
I saw that we didn't for creation either so I maintained that. I'm open to adding it.

Reply:
@TheCleric the rationale to skip platforms/archs at creation was to avoid slowing down tests too much.

How slow are these extra tests?

- If they're not too slow, I'm fine adding Windows to exercise your WinPE detection code, for exhaustiveness.
- On the contrary, if they are slow, I'm open to cutting down this list even more to our most important couples (e.g. sacrificing testing linux / ia32,arm,arm64).

Reply:
> How slow are these extra tests?

Somewhere in the realm of 2x as slow as the creation tests as we have to create an app package, and then "upgrade" from it and validate it pulled the correct params from the "old" app. So essentially calling create twice.

On my box the integration tests took **14** seconds before and take @ **114** seconds now.

So I'll defer to your judgement on what's best. 🤣

Reply:
Hm, yeah, not a fan of lengthening these tests so much. I pushed a commit limiting the tests to two couples.